modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
+or `stable-{product-version}` channels to the latest release in {product-version}.z,
+although there can be delays while safe paths away from troubled releases are
 constructed and verified.
 
 [discrete]
-== Fast and Stable Channel Usage and Strategies
+== Fast and stable channel use and strategies
 
-The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+The `fast-{product-version}` and `stable-{product-version}` channels present a choice between receiving
 general availability releases as soon as they are available or allowing Red Hat to
 control the rollout of those updates. If issues are detected during rollout or at a
-later time, upgrades to that version may be blocked in both the fast-{product-version} and
-stable-{product-version} channels, and a new version may be introduced that will be the new
+later time, upgrades to that version might be blocked in both the `fast-{product-version}` and
+`stable-{product-version}` channels, and a new version might be introduced that becomes the new
 preferred upgrade target.
 
 Customers can improve this process by configuring pre-production systems on the
-fast-{product-version} channel, production systems on the stable-{product-version} channel,
-and participating in Red Hat’s xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc[connected customer program]. This program allows Red
+`fast-{product-version}` channel, configuring production systems on the `stable-{product-version}` channel,
+and participating in Red Hat’s connected customer program. This program allows Red
 Hat to observe the impact of updates on your specific hardware and software
-configurations. Future releases may improve or alter the pace at which updates move
-from the fast-{product-version} to the stable-{product-version} channel.
+configurations. Future releases might improve or alter the pace at which updates move
+from the `fast-{product-version}` to the `stable-{product-version}` channel.
 
 [discrete]
-== Restricted Network Clusters
+== Restricted network clusters
 
-Customers who have chosen to not be connected to Red Hat and are controlling their
-own {product-title} container image content manually should consult the Red Hat
-errata associated with product releases and note any comments impacting upgrades.
-During upgrade the user interface may caution about switching between these versions
-and it is up to the customer to ensure they have correctly selected the appropriate
-version before bypassing those cautions.
+If you manage the container images for your {product-title} clusters yourself, you must consult the Red Hat errata that is associated with product releases and note any comments that impact upgrades.
+During upgrade, the user interface provides warnings about updating to versions that contain
+vulnerabilities, so you must ensure that you selected an appropriate version before bypassing those warnings.
```

Comment:
"that contain vulnerabilities" is too specific, so ->

"During upgrade, the user interface provides warnings about updating to **certain versions**, so you must ensure that you selected an appropriate version before bypassing those warnings."

Reply:
This section is not about vulnerabilities, it's about:

> If your cluster cannot reach it's upstream Cincinnati service (e.g. our Red Hat Cincinnati service), it will not know about any recommended updates, so any update will need the admin to say:
> > It's ok, cluster-version operator, I'm comfortable with this update despite you never having heard of it before.

I think the previous master wording was closer to this behavior. Can we make smaller pivots to address any style issues with the master language?